apps/admin-portal/src/api/user-stores.ts

```diff
@@ -26,6 +26,11 @@ import { AxiosHttpClient } from "@wso2is/http";
  */
 const httpClient = AxiosHttpClient.getInstance();
 
+/**
+ * Fetches all user stores
```

Comment:
Please keep spaces between method comment, param, and return. Add full stops as well

Reply:
Check other places as well